src/trans.ts

```diff
@@ -1,65 +1,67 @@
-import { get, reduce, reject } from 'lodash';
-import { OPTIONS } from './i18n-json-webpack-loader';
+import { get, reduce } from 'lodash';
 
-const isCreateElement = (entity) => get(entity, 'callee.property.name') === 'createElement';
+export const isElement = entity =>
+  get(entity, 'type') === 'CallExpression' &&
+  get(entity, 'callee.property.name') === 'createElement';
 
-const checkTransOn = (obj) => (
-  get(obj, 'type') === 'MemberExpression'
```

Comment:
What was the reason for dropping this check?

Reply:
I didn't understand what it was doing (as distinct from the `isTrans` helper), and once `isTrans` was rewritten to be a bit more general it did not seem to be needed. If there was a special case that it covered, we should capture that in a test.